crawler/twitter.js

```diff
@@ -4,9 +4,12 @@ const cfg       = require('../core/config'),
       debug     = require('../core/debug')('crawler/twitter', true),
       Character = require('../models/character'),
       Tweet     = require('../models/tweet'),
-      Twit      = require('twit');
+      Twit      = require('twitter'),
```

Comment:
Please don't change the twitter client back. I changed it for a reason. Neither of both is an official package, but the latter is better maintained.